### PR TITLE
[COLAB-2453] Fix twitter preview image

### PR DESCRIPTION
**Pull request by jobachhu**
This PR fixes the proposal logo URL generation (copying the method used for the contest logo). The proposal logo was still relative and Twitter requires absolute URLs.

---
This change is [Reviewable](https://reviewable.io/reviews/cci-mit/xcolab/88)

### CODE REVIEW

File: view/src/main/webapp/WEB-INF/jsp/proposals/proposalDetails_edit.jspx [hunk 1]
```diff
@@ -114,12 +114,6 @@
         </div> <!-- /c-Headline -->
 
 
-        <form action="${contest.proposalLogoPath}images/upload" method="post" enctype="multipart/form-data" target="_fileUploadFrame" id="fileUploadForm">
-            <input type="hidden" name="${_csrf.parameterName}" value="${_csrf.token}"/>
-            <input type="file" name="file" id="fileUploadInput" />
-        </form>
-
-
```
Comment (umov) [inline view/src/main/webapp/WEB-INF/jsp/proposals/proposalDetails_edit.jspx]: Is this change related?

Comment (jobachhu) [reply view/src/main/webapp/WEB-INF/jsp/proposals/proposalDetails_edit.jspx]: Indirectly - it was using the method that I changed. This code would have caused an error after changing the method, but since it's unused I removed it.

Comment (umov) [reply view/src/main/webapp/WEB-INF/jsp/proposals/proposalDetails_edit.jspx]: Okay, got it.
